src/SOS/Strike/stressLogDump.cpp

```diff
@@ -307,6 +307,30 @@ vDoOut(BOOL bToConsole, FILE* file, PCSTR Format, ...)
     va_end(Args);
 }
 
+static TADDR GetFormatAddr(StressLog& inProcLog, uint32_t formatOffset)
+{
+    // does the module table look valid?
+    if (inProcLog.moduleOffset == (size_t)inProcLog.modules[0].baseAddress &&
```

Comment:
You could check or also check the SOS breaking change version here. The call to CheckBreakingRuntimeChange() can return it, but plumbing it through to this function is messy.  If you think these checks are good enough then don't worry about it.

Reply:
It doesn't seem hard to check the SOS breaking change version - I have updated the PR do that.